packages/server-core/src/social/channel/channel.hooks.ts

```diff
@@ -111,7 +111,8 @@ const ensureUserHasChannelAccess = async (context: HookContext<ChannelService>)
  * @returns
  */
 const ensureUsersFriendWithOwner = async (context: HookResolver<ChannelService, ChannelData>) => {
-  const users = context.data.users
+  const [data] = Array.isArray(context.data) ? context.data : [context.data]
```

Comment:
This will work for 1st entry in `context.data` only. As we are allowing default `.class` file now, which allows user to send data as multiple records also.
i.e.
![image](https://github.com/EtherealEngine/etherealengine/assets/10975502/06212889-7e02-4351-a955-3578c4f9fbdc)